rpc/handlers.go

```diff
@@ -1,11 +1,13 @@
 package rpc
 
 import (
+	"context"
 	"encoding/json"
 	"errors"
 	"strings"
 
 	"github.com/NethermindEth/juno/blockchain"
+	"github.com/NethermindEth/juno/clients/feeder"
```

Comment:
To be consistent, let's define a interface like we did with `Gateway` do avoid importing this package.

Reply:
We would still need this package because there is a reference to type `TransactionStatus`:
```
type Feeder interface {
	Transaction(context.Context, *felt.Felt) (*feeder.TransactionStatus, error)
}
```
Do you think I need to use interface anyway?

Reply:
I dont think we do in that case.

But @IronGauntlets might have stronger opinions in this matter. I will leave it to him.

Reply:
I think for now it is fine. If we were to introduce the interface here we should do it for `Starknet` data as well, which is out of scope of the pr.